dockstore-webservice/src/main/java/io/dockstore/webservice/core/DescriptorTypeConverter.java

```diff
@@ -0,0 +1,37 @@
+package io.dockstore.webservice.core;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import javax.persistence.AttributeConverter;
+import javax.persistence.Converter;
+
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Converter(autoApply = true)
+public class DescriptorTypeConverter implements AttributeConverter<List<String>, String> {
```

Comment:
Is this converting between a text column in the DB to an array? 
There is a native text array that might be more efficient 
example here https://www.postgresqltutorial.com/postgresql-array/

Reply:
yeah that's what it's doing. I was looking around, but I was reading hibernate doesn't natively support an array type column in the db. That it only accepts mapping lists and sets to tables, but maybe not the case anymore? The examples I've seen online add other maven dependencies to so.

Reply:
Yeah, it looks like hibernate doesn't support it natively. 
I think I can live with this then